src/main/java/com/conveyal/datatools/manager/jobs/ProcessSingleFeedJob.java

```diff
@@ -180,17 +181,36 @@ public void jobLogic () {
         }
     }
 
+    /**
+     * Once the job is complete, notify subscribers and provide feedback on the job creation status.
+     */
     @Override
     public void jobFinished () {
+        String message;
         if (!status.error) {
             // Note: storing a new feed version in database is handled at completion of the ValidateFeedJob subtask.
             status.completeSuccessfully("New version saved.");
+            if (feedVersion.validationResult.errorCount > 0) {
+                message = String.format("New feed version created for %s between %s - %s. During validation, we found %s issue(s)",
+                    feedSource.name,
+                    feedVersion.validationResult.firstCalendarDate,
+                    feedVersion.validationResult.lastCalendarDate,
+                    feedVersion.validationResult.errorCount);
+            } else {
+                message = String.format("New feed version created for %s. The validation check found no issues with this new dataset!", feedSource.name);
```

Comment:
Would be nice to have the date range for versions without errors too.

Reply:
Updated.